CMakeListsGtestInclude.cmake

```diff
@@ -0,0 +1,33 @@
+#
+# Copyright (C) 2019 Draios Inc dba Sysdig.
```

Comment:
Nit: We're not Draios anymore, I suggest for these new files you just make them say Sysdig.